lib/voorhees/json_api.ex

```diff
@@ -55,17 +55,41 @@ defmodule Voorhees.JSONApi do
   defp error_message(:ok), do: ""
   defp error_message({:error, message}), do: "Payload did not match expected\n\n" <> message
 
+  defp compare_property(actual, expected, property_name, options) do
+    actual_value = Map.fetch(actual, property_name)
+    expected_value =  Map.fetch(expected, property_name)
+
+    case {actual_value, expected_value} do
+      {:error, :error} -> :ok
+      {:error, _} -> compare_resources(actual_value, expected_value, options)
```

Comment:
Again, use pattern matching here, then you can put the fetch in the case statement on 62